src/commands.rs

```diff
@@ -250,16 +251,40 @@ impl AssetCommand {
         Ok(Assets::new(get(&self.authentication, &endpoint)?))
     }
 
+    fn add_web_asset(&self, url: &str, headers: &HeaderMap, payload: &HashMap<&str, &String>) -> anyhow::Result<Assets, CommandError> {
+        let response = self
+            .authentication
+            .build_client()?
+            .post(url)
+            .json(payload)
+            .headers(headers.clone())
+            .send()?;
+
+        if response.status().as_u16() != 201 {
+            let status = response.status().as_u16();
+            return Err(CommandError::WrongResponseStatus(status));
+        }
+
+        Ok(Assets::new(serde_json::from_str(&response.text()?)?))
+    }
+
     pub fn add(&self, path: String, title: String) -> anyhow::Result<Assets, CommandError> {
         let url = format!("{}/v4/assets", &self.authentication.config.url);
 
         let mut headers = HeaderMap::new();
         headers.insert("Prefer", "return=representation".parse()?);
 
+        if path.starts_with("http") || path.starts_with("https") {
```

Comment:
isn't path.starts_with("http") enough? path.starts_with("https") evaluated to true - means path.starts_with("http") also will be evaluated to true.

Reply:
true! though I guess I need to do http: and https: because local files can start with http as well.